openslides_backend/action/actions/motion/import_.py

```diff
@@ -0,0 +1,644 @@
+                    create_action_payload.append(payload)
+                    errors = self.get_create_payload_integrity_error_message(
+                        payload, meeting_id
+                    )
+                else:
+                    id_ = payload["id"]
+                    motion_to_submitter_user_ids[id_] = submitters
+                    motion = {
+                        k: v
+                        for k, v in (
+                            [
+                                motion
+                                for motion in self.number_lookup.name_to_ids.get(
+                                    payload["number"], []
+                                )
+                                if motion.get("id")
+                            ][0]
+                        ).items()
+                    }
+                    for field in ["category_id", "block_id"]:
+                        if payload.get(field) is None:
+                            if not motion.get(field):
+                                payload.pop(field, None)
+                    if len(submitters):
+                        motion_submitter_ids: list[int] = (
+                            motion.get("submitter_ids", []) or []
+                        )
+                        matched_submitters = {
+                            self._submitter_ids_to_user_id[submitter_id]: submitter_id
+                            for submitter_id in motion_submitter_ids
+                            if self._submitter_ids_to_user_id.get(submitter_id)
+                            in submitters
+                        }
+                        submitter_create_action_payload.extend(
+                            [
+                                {
+                                    "meeting_user_id": self._user_ids_to_meeting_user[
+                                        user_id
+                                    ]["id"],
+                                    "motion_id": id_,
+                                }
+                                for user_id in submitters
+                                if user_id not in matched_submitters.keys()
+                            ]
+                        )
+                        submitter_delete_action_payload.extend(
+                            [
+                                {"id": submitter_id}
+                                for submitter_id in motion_submitter_ids
+                                if submitter_id not in matched_submitters.values()
+                            ]
+                        )
+                        old_submitters[id_] = matched_submitters
+
+                    payload.pop("meeting_id", None)
+                    update_action_payload.append(payload)
+                    errors = self.get_update_payload_integrity_error_message(
+                        payload, meeting_id
+                    )
+                for err in errors:
+                    fieldname = ""
```

Comment:
Maybe the problem will be resolved by other tests, but  there is no test with errors passing this code